include/zephyr/drivers/disks/sdmmc_stm32.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2025 The Zephyr Contributors.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_DISK_SDMMC_STM32_H_
+#define ZEPHYR_DRIVERS_DISK_SDMMC_STM32_H_
```

Comment:
Since it's in the generic *include/zephyr/* path, this should be renamed to
`ZEPHYR_INCLUDE_DRIVERS_DISKS_SDMMC_STM32_H_`.